src/banana/data/genpdf/__init__.py

```diff
@@ -16,11 +16,30 @@ def generate_pdf(
     """
     Generate a new PDF from a parent PDF with a set of flavors.
 
+    If parent_pdf_set is the name of an available PDF set, it will be used as parent. In order to use
+    the toy PDF as parent, it is enough to set parent_pdf_set = "toy" or "toylh".
+    If parent_pdf_set is not specified, a debug PDF constructed as x * (1-x) for every flavor will be used
+    as parent.
+    It is also possible to provide custom functions for each flavor in the form of a dictionary: {pid: f(x,Q2)}.
+
+    In labels it is possible to pass a list of PIDs or evolution basis combinations to keep in the generated PDF.
+    In order to project on custom combinations of PIDs, it is also possible to pass a list containing the desired
+    factors for each flavor.
+
+    The default behaviour is to generate only one member for a PDF set (the zero member) but it can be changed setting
+    to True the member flag.
+
+    The info_update argument is a dictionary and provide to the user a way to change the info file of the
+    generated PDF set. If a key of info_update matches with one key of the standard info file, the information
+    are updated, otherwise they are simply added.
+
+    Turning True the value of the install flag, it is possible to autmatically install the generated PDF to
+    the lhapdf directory. By default install is False.
     Parameters
     ----------
         name : str
             target name
-        labels : list(int)
+        labels : list(str)
```

Comment:
don't put a type here, since it can be at least 3 different ones (and their not compatible with each other)